### DISCUSSION TITLE
Chat macro/copy paste

### DISCUSSION THREAD

**Discussion by liiinder** [SnosMe/awakened-poe-trade#1286] [Ideas]
Hey!

Would it be possible to fix in a future update so your current copy paste is restored after you do a /exit /hideout? Feels like that shouldn't be a hard thing to implement :)

**Comment by SnosMe** [comment]
It's already in :)

![image](https://github.com/SnosMe/awakened-poe-trade/assets/4292308/82993e6c-7f2e-4877-8ed4-045d60186053)

**Reply by liiinder** [reply]
🤯 Is it now or why isn't this default? 😅

**Reply by SnosMe** [reply]
It was by default... not anymore because not careful use of it can leak clipboard #742

**Reply by liiinder** [reply]
Ah okay! That makes sense 👍 Thanks alot!
